app/components/graph-editor/defaults.ts

```diff
@@ -99,20 +97,17 @@ export const NUDGE: number = 3;
 export const STICKY_DELAY: number = 500;
 
 /**
- * NODE_DEFAULTS  
+ * NODE_PROPERTIES  
  *   Default property values for drawable nodes.
- * 
- *   Only `DrawableNode` properties should be in this object and nothing else.
- *   This object is used as the template for `isDrawableNode`.
- */
-export const NODE_DEFAULTS: DrawableNode = {
-    position: { x: 0, y: 0 },
-    label: "",
-    shape: "circle",
-    color: "#fff200",
-    borderColor: "#000",
-    borderStyle: "solid",
-    borderWidth: 1
+ */
+export const NODE_PROPERTIES = {
```

Comment:
I agree. Currently this entire file is a stand-in for what should be a global user settings file, except that it currently only contains stuff related to the graph editor.